airbyte-webapp/src/components/ui/Callout/Callout.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from "classnames";
+
+import styles from "./Callout.module.scss";
+
+interface CalloutProps {
+  className?: string;
+  variant?: "yellow" | "red" | "blue";
+}
+
+export const Callout: React.FC<React.PropsWithChildren<CalloutProps>> = ({
+  children,
+  className,
+
+  variant = "yellow",
+}) => {
+  const containerStyles = classNames(styles.container, className, {
+    [styles.yellow]: variant === "yellow",
+    [styles.red]: variant === "red",
+    [styles.blue]: variant === "blue",
+  });
+
+  return <div className={containerStyles}>{children}</div>;
```

Comment:
Take it or leave it: should we add a prop for an optional icon? I imagine a lot of callouts have an icon, and that would make it easy to ensure spacing between icon & text (or whatever is in `{children}`) is consistent.

Reply:
IMO no.  I took out the existing prop for that as it was only used in 1-2 places.  

There is also a spot where I plan on adding a button to a Callout, etc.  Once I started considering different ways this _might_ be used, I ended up deciding the important part here is really just the basic container.

I'm more in favor of keeping this very basic and unconcerned with what its children are personally.  It does have flex, so it does space out its children consistently by default (which can be overridden by a classname if wanted)

Reply:
Having said that I just saw one more usage when I rebased.

However, with the move away from FA towards custom icons, we'd likely be passing in a ReactNode or a svg, at which point it feels to me like we may as well just accept it as a child... I could be swayed there though.

Reply:
Fine for me to skip this for now! I think when we do eventually move towards custom icons, we'll still want to enforce sizing constrains so that different `<Callout>` usages don't have different icon sizes, something like `<Callout icon="exclamation-mark">{ ... some text or JSX ... }</Callout>`

But no need to address that yet - we can probably enforce that later if/when we implement a custom `Icon` component: https://github.com/airbytehq/airbyte/issues/20133